tutorial/10_key_features/005_visualization.py

```diff
@@ -53,6 +53,14 @@
 from optuna.visualization import plot_slice
 from optuna.visualization import plot_timeline
 
+try:
+    import optuna_integration
+
+    is_integration_available = True
+except ImportError:
+    is_integration_available = False
```

Comment:
Generally, we should not raise an exception in docs because it results in [a documentation build error](https://github.com/optuna/optuna/actions/runs/7897750862/job/21553864779?pr=5249).

I feel that installing `optuna_integration` in the CI for the documentation build is the simplest way to resolve all issues.